### PR TITLE
Abdul/iot hub clients

### PR DESCRIPTION
**Pull request by v-abdrahman**
Created a folder for IoTHubForegroundClient app & added all the project files for windows 10 IoT Enterprise.

Following changes are done:

1) file modified : IoTHubForegroundClient.csproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.

2) file modified : README.md
    Adding windows 10 iot Enterprise in products.
    Updated the Windows Platform from IoT Core to IoT Enterprise
    Added the app *packages installing steps

### CODE REVIEW

File: IoTHubClients/CS/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+#### In this article
+
+This article describes how, as a device developer, to connect a device running a Windows 10 IoT Enterprise device (like Up-Board) to your Microsoft IoT Central application using the C# programming language.
+
+### Before you begin
+
+To complete the steps in this article, you need the following:
+
+- A Microsoft IoT Central application created from the Sample Devkits application template. For more information, see [Create your Microsoft IoT Central Application](https://docs.microsoft.com/en-us/microsoft-iot-central/howto-create-application).
+- A device running the Windows 10 IoT Enterprise system.  For this walkthrough, we will use a Up-Board.
```
Comment (neilsh-msft) [inline IoTHubClients/CS/README.md]: "a Up-Board" should be "an Up-Board"

File: IoTHubClients/CS/IoTHubForegroundClient.csproj [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
+    <ProjectGuid>{48C3219E-C4A2-468A-8AD5-4911E50B6F45}</ProjectGuid>
+    <OutputType>AppContainerExe</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>IoTHubForegroundClient</RootNamespace>
+    <AssemblyName>IoTHubForegroundClient</AssemblyName>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
+    <TargetPlatformVersion>10.0.17763.0</TargetPlatformVersion>
```
Comment (neilsh-msft) [inline IoTHubClients/CS/IoTHubForegroundClient.csproj]: Should this value be updated from 17763 to 19041?

Comment (v-abdrahman) [reply IoTHubClients/CS/IoTHubForegroundClient.csproj]: Updated the PlatformVersion to 19041 and PlatformMinVersion to 15063

File: IoTHubClients/CS/README.md (hunk 1 above)
Comment (v-abdrahman) [reply IoTHubClients/CS/README.md]: Made the changes as mentioned in the readme document

**Pull request merged**